packages/react-core/src/components/AlertGroup/examples/AlertGroup.md

```diff
@@ -6,37 +6,68 @@ propComponents: ['Alert', 'AlertGroup', 'AlertActionCloseButton', 'AlertActionLi
 ---
 
 ## Examples
+
+### Variant examples
+
+Alert groups utilize the same variants that are available to [alert components](components/alert/). You can add multiple alert variants to each group.
+
 ### Static alert group
-These alerts appear on page load and are discoverable from within the normal page content flow, and will not be announced individually/explicitly to assistive technology.
+
+Static alert groups appear when the page loads. They are seen within the normal page content flow. They will not be announced individually or explicitly to assistive technology.
+
 ```ts file="./AlertGroupStatic.tsx"
 ```
 
-### Toast alert group
-Alerts asynchronously appended into dynamic AlertGroups with `isLiveRegion` will be announced to assistive technology at the moment the change happens, following the strategy used for aria-atomic, which defaults to false. This means only changes of type "addition" will be announced.
+### Toast alert groups
+
+Toast alerts appear in groups on top of content at the top right of the page. Click the buttons in the example below to simulate toast alerts on this webpage.
```

Comment:
Something along the lines of the following, to make it clear exactly what a toast alert group is doing (compared to a dynamic, non-toast alert group)

```suggestion
Toast alert groups position alerts on top of other content at the top right of the page. Click the buttons in the example below to simulate toast alerts on this webpage.
```